### PR TITLE
K8SPXC-485: Add log collector to sources

### PR DESCRIPTION
**Pull request by blez**

### CODE REVIEW

File: sources/operator.1.4.0.pxc-operator.json [hunk 1]
```diff
@@ -54,6 +54,14 @@
             "critical": false
           }
         },
+        "log_collector": {
```
Comment (delgod) [inline sources/operator.1.4.0.pxc-operator.json:57]: just Q: is it mandatory to add a new image to non-relevant versions?

**Review by delgod — CHANGES_REQUESTED** [review]

File: sources/operator.1.4.0.pxc-operator.json (hunk 1 above)
Comment (blez) [reply sources/operator.1.4.0.pxc-operator.json:57]: I though log collector can be used with any version of PXC.No?

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | passed |

File: sources/operator.1.4.0.pxc-operator.json (hunk 1 above)
Comment (delgod) [reply sources/operator.1.4.0.pxc-operator.json:57]: with any version of PXC - yes.
but only with >= 1.7.0 version of Operator

Comment (blez) [reply sources/operator.1.4.0.pxc-operator.json]: https://github.com/Percona-Lab/percona-version-service/pull/26/commits/2ce22432b1ed32810692dd51675efb571f088b98

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | passed |

**Review by delgod — APPROVED** [review]

**Review by qJkee — APPROVED** [review]

**Review by tplavcic — APPROVED** [review]

**Pull request merged**